CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 * [BUGFIX] Memberlist: Entry in the ring should now not appear again after using "Forget" feature (unless it's still heartbeating). #3603
 * [BUGFIX] Ingester: do not close idle TSDBs while blocks shipping is in progress. #3630
 * [BUGFIX] Ingester: correctly update `cortex_ingester_memory_users` and `cortex_ingester_active_series` when a tenant's idle TSDB is closed, when running Cortex with the blocks storage. #3646
+* [BUGFIX] Worker: fix default incorrectly overriding FrontendAddress. #3650
```

Comment:
```suggestion
* [BUGFIX] Querier: fix default value incorrectly overriding `-querier.frontend-address` in single-binary mode.  #3650
```

Reply:
I guess I was confused by the file naming(`pkg/querier/worker/worker.go`) and config section name `frontend_worker`.

Reply:
No worries, it *is* confusing. Querier is the top-level Cortex component, which internally runs "frontend worker" that connects to frontend component. Thanks for your changes!